Sources/Bob/Commands/Info/InfoCommand.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  InfoCommand.swift
+//  Bob
+//
+//  Created by Jan Chaloupecky on 02.12.19.
+//
+
+import Foundation
+
+
+/// The `InfoCommand` prints a list of key/values provided by `InfoProvider`
+/// It's meant to provide info about a running instance of Bob
+public class InfoCommand: Command {
+    private var providers: [InfoProvider] = [
+        InstanceIdInfoProvider(),
+        UptimeInfoProvider()
```

Comment:
Nope, The `EnvironmentInfoProvider` is meant to be setup during the the `boot` so that users can select the env values they want to be part of the output